.github/workflows/Webhook-Cleanup.yml

```diff
@@ -0,0 +1,26 @@
+name: Webhook Requests Cleanup
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Is it necessary to run clean up every midnight? As people are working from different time zones it is hard to say what is the right midnight for clean up. I would suggest to do clean up only for example at Saturday midnight when there is the minimal risk that something will be cleaned up for example during the PR check run. See https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/.github/workflows/CI-Topics-Cleanup.yml#L5

Reply:
We are deleting requests which are older than 24 hours, so it shouldn't affect any test execution.

Reply:
Understand. Thank you for the explanation. :+1: